browser/admin/src/integrator/AdminIntegratorSettings.ts

```diff
@@ -325,15 +325,22 @@ class SettingIframe {
 		formData.append('type', this.getConfigType());
 
 		try {
-			const response = await fetch(this.API_ENDPOINTS.fetchSharedConfig, {
-				method: 'POST',
-				headers: {
-					Authorization: `Bearer ${window.accessToken}`,
+			const response: Response = await fetch(
+				this.API_ENDPOINTS.fetchSharedConfig,
+				{
+					method: 'POST',
+					headers: {
+						Authorization: `Bearer ${window.accessToken}`,
+					},
+					body: formData,
 				},
-				body: formData,
-			});
+			);
 
 			if (!response.ok) {
+				console.error(
+					'something wend wrong shared config response',
+					response.text(),
```

Comment:
There is a typo here. This is anyhow console error but would be good to fix it I think as another PR

Reply:
wend -> went